src/app/modules/dogs/store/reducer.ts

```diff
@@ -1,12 +1,13 @@
-import { Action } from "rxjs/internal/scheduler/Action";
 import { DogActions } from "./actions";
 import { DogsState, initialDogState } from "./state";
 
 export function dogsReducer(state: DogsState = initialDogState, action: DogActions): DogsState {
   if (!action) return state;
   switch (action.type) {
-    case '[dogs] setDogs':
+    case '[dogs] setAllDogs':
       return { ...state, dogs: action.dogs };
+    case '[dogs] getAllDogs':
+      return { ...state, ongoingActions: [ ...state.ongoingActions, action.type ] };
```

Comment:
Here it looks good, but if there is more changes consider breaking it into multiple lines to improve readability